test/resources/demo-users/user/kustomization.yaml

```diff
@@ -3,4 +3,6 @@ kind: Kustomization
 resources:
   - ns1
   - ns2
+  - managed-ns1
+  - managed-ns2
```

Comment:
At least for `managed-ns2`, that resource contains the RPA which needs to be edited by the user before it's deployed to the cluster.
the way things are deployed needs to change a bit to keep things coherent.

Reply:
Ah, I see. Ok, then we can drop this.